containers/proxy-systemd-services/uyuni-proxy-salt-broker.service

```diff
@@ -13,9 +13,16 @@ After=uyuni-proxy-pod.service
 Environment=PODMAN_SYSTEMD_UNIT=%n
 EnvironmentFile=/etc/sysconfig/uyuni-proxy-systemd-services
 EnvironmentFile=/etc/sysconfig/proxy
+EnvironmentFile=/etc/sysconfig/uyuni-proxy-salt-broker-service
```

Comment:
Are you sure this will not trigger error if file does not exists?
According to [documentation](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=):

>The argument passed should be an absolute filename or wildcard expression, optionally prefixed with "-", which indicates that if the file does not exist, it will not be read and no error or warning message is logged.

Reply:
Oh right, you've mentioned it above. So it should be 

`EnvironmentFile=-/etc/sysconfig/uyuni-proxy-salt-broker-service`